templates/sshd_config.erb

```diff
@@ -76,7 +76,9 @@ AuthorizedKeysCommandUser <%= @sshd_authorized_keys_command_user %>
 # For this to work you will also need host keys in /etc/ssh/ssh_known_hosts
 #RhostsRSAAuthentication no
 # similar for protocol version 2
-#HostbasedAuthentication no
+<% if @sshd_hostbasedauthentication -%>
```

Comment:
Missed this one. We don't need the conditional logic here for the same reason as the manifest as the value must be 'yes' or 'no' and cannot be `undef`.